package.json

```diff
@@ -16,7 +16,8 @@
   "gypfile": true,
   "scripts": {
     "install": "node-gyp rebuild",
-    "pretest": "jscs . && jshint .",
+    "instrument": "node node_modules/tap/node_modules/nyc/node_modules/istanbul/lib/cli.js instrument --output backend.cov --no-compact --preserve-comments backend",
```

Comment:
This is rather ugly, but I'd rather have a build that fails because the dependency tree changed, than to have a subtly broken code coverage report caused by running a different istanbul version for instrumentation and report generation.

And yes, I do realise I'll have to rework this anyway after we add a `npm@3` CI slave.